src/MPAGSCipher/CipherMode.hpp

```diff
@@ -0,0 +1,9 @@
+#ifndef CIPHERMODE_HPP
+#define CIPHERMODE_HPP
+
+enum class CipherMode{
+    decrypt,
+    encrypt
```

Comment:
Our naming convention specified UpperCamelCase for `enum` states:
```suggestion
    Decrypt,
    Encrypt
```